pages/Filter/index.jsx

```diff
@@ -84,13 +84,18 @@ class Filter extends Component {
           }, 300),
         };
       }
-      default:
       case FILTER_TYPE_MULTISELECT:
         return {
           ...filter,
           url: `${filter.url}${this.props.queryParams}`,
           active: temporaryFilter ? temporaryFilter.values : null,
         };
+      default:
```

Comment:
why did you change it? it's still the same code for both cases

Reply:
no it's not the same code (value != values)